inbm/dispatcher-agent/dispatcher/sota/rebooter.py

```diff
@@ -30,6 +30,9 @@ def reboot(self) -> None:
         logger.debug("")
         self._dispatcher_broker.telemetry("Rebooting ")
         time.sleep(2)
+        
+    def is_reboot(self, device_reboot: str) -> bool:
```

Comment:
Consider moving the `is_reboot` method to a common utility class or module to avoid duplication across different classes that need to check the reboot condition. [important]